app/src/main/kotlin/com/hippo/ehviewer/ui/tools/Dialog.kt

```diff
@@ -188,10 +191,9 @@ class DialogState {
             title = title.ifNotNullThen { Text(text = title!!) },
             text = {
                 OutlinedTextField(
-                    value = state,
-                    onValueChange = { state = it },
-                    label = hint.ifNotNullThen {
-                        Text(text = hint!!)
+                    state = state,
+                    label = hint?.let {
+                        @Composable { Text(text = it) }
```

Comment:
This `@Composable` cannot be inferred?

Reply:
It shows as an error in AS, but seems to [compile successfully](https://github.com/FooIbar/EhViewer/actions/runs/10268001500)?